eternalcore/src/test/java/com/eternalcode/core/user/PrepareUserControllerTest.java

```diff
@@ -0,0 +1,65 @@
+package com.eternalcode.core.user;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.MockPlugin;
+import be.seeseemelk.mockbukkit.ServerMock;
+import be.seeseemelk.mockbukkit.entity.PlayerMock;
+import com.eternalcode.core.user.client.ClientBukkitSettings;
+import com.eternalcode.core.user.client.ClientSettings;
+import net.kyori.adventure.text.Component;
+import org.bukkit.event.player.PlayerKickEvent;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import panda.std.Option;
+import panda.utilities.StringUtils;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PrepareUserControllerTest {
+
+    private static ServerMock server;
+    private static MockPlugin plugin;
+
+    @BeforeAll
+    public static void load() {
+        server = MockBukkit.mock();
+        plugin = MockBukkit.createMockPlugin();
```

Comment:
```suggestion
        this.server = MockBukkit.mock();
        this.plugin = MockBukkit.createMockPlugin();
```

Z tego co widzę, to `server` oraz `plugin` są odwołaniami do fieldów. Wypadałoby oznaczyć je słowami `this`.

Reply:
to jest static field